modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc

```diff
@@ -7,46 +7,88 @@
++
+|===
+|To... |Do this...
 
-. Select *Insert Metric at Cursor* to view a list of predefined queries.
+|Insert a predefined query. |Select *Insert Metric at Cursor* to view a list of predefined queries.
 
-. To create a custom query, add your Prometheus Query Language (PromQL) query to the *Expression* field.
-+
-[NOTE]
-====
-As you type a PromQL expression, autocomplete suggestions appear in a drop-down list.
-These suggestions include functions, metrics, labels, and time tokens.
-You can use the keyboard arrows to select one of these suggested items and then press Enter to add the item to your expression.
-You can also move your mouse pointer over a suggested item to view a brief description of that item.
-====
+|Create a custom query. 
+|Add your Prometheus Query Language (PromQL) query to the *Expression* field.
 
-. To add multiple queries, select *Add Query*.
+As you type a PromQL expression, autocomplete suggestions appear in a drop-down list. These suggestions include functions, metrics, labels, and time tokens.
+You can use the keyboard arrows to select one of these suggested items and then press Enter to add the item to your expression. You can also move your mouse pointer over a suggested item to view a brief description of that item.
 
-. To duplicate an existing query, select {kebab} next to the query, then choose *Duplicate query*.
+|Add multiple queries. |Select *Add Query*.
 
-. To delete a query, select {kebab} next to the query, then choose *Delete query*.
+|Duplicate an existing query. |Select {kebab} next to the query, then choose *Duplicate query*.
 
-. To disable a query from being run, select {kebab} next to the query and choose *Disable query*.
+|Disable a query from being run. |Select {kebab} next to the query and choose *Disable query*.
+|===
 
-. To run queries that you created, select *Run Queries*. 
-The metrics from the queries are visualized on the plot. 
-If a query is invalid, the UI shows an error message.
+. To run queries that you created, select *Run Queries*. The metrics from the queries are visualized on the plot. If a query is invalid, the UI shows an error message.
 +
 [NOTE]
 ====
 Queries that operate on large amounts of data might time out or overload the browser when drawing time series graphs. To avoid this, select *Hide graph* and calibrate your query using only the metrics table. Then, after finding a feasible query, enable the plot to draw the graphs.
 ====
++
+[NOTE]
+====
+By default, the query table shows an expanded view that lists every metric and its current value. You can select *˅* to minimize the expanded view for a query.
+====
 
 . Optional: The page URL now contains the queries you ran. To use this set of queries again in the future, save this URL.
+
+. Explore the visualized metrics. Initially, all metrics from all enabled queries are shown on the plot. You can select which metrics are shown by doing any of the following:
++
+|===
+|To... |Do this...
```

Comment:
https://github.com/openshift/openshift-docs/pull/60762/files#r1214688029

Reply:
I changed the column headings to "Option" and "Description".